components/filter-field/src/filter-field-range/filter-field-range-trigger.ts

```diff
@@ -139,6 +153,11 @@ export class DtFilterFieldRangeTrigger implements OnDestroy {
     private _changeDetectorRef: ChangeDetectorRef,
     zone: NgZone,
     renderer: Renderer2,
+    @Optional() private _viewportRuler: ViewportRuler,
```

Comment:
to make it non breaking pls make the prop itself also optional, not only via DI.
```suggestion
    @Optional() private _viewportRuler?: ViewportRuler,
```

Also annotate it as a breaking fo 7.0.0 to remove it again